Tests/LibWeb/Ref/reference/scrolled-text-with-clipped-corners-ref.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<link rel="match" href="reference/scrollable-contains-boxes-with-hidden-overflow-2-ref.html" />
```

Comment:
What is this a reference to? :thinking:

Reply:
ops, that's something left from another test I used as a template. fixed!